test_runner/regress/test_remote_storage.py

```diff
@@ -607,7 +607,8 @@ def assert_compacted_and_uploads_queued():
         ".* ERROR .*Error processing HTTP request: InternalServerError\\(timeline is Stopping"
     )
 
-    timeline_delete_wait_completed(client, tenant_id, timeline_id)
+    # Generous timeout, because currently deletions can get blocked waiting for compaction
```

Comment:
I'd rather try to keep timeout small to not slow down tests. If this isnt enough because of the compaction lets link the issue there and mention in the comment that when the issue is resolved we can reduce the timeout. WDYT?

Reply:
Yep, added a ticket for this behaviour and linked in the comment https://github.com/neondatabase/neon/pull/4992/commits/a70af8037f728af4a97a1f89f978a5389cdbc1ba